src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
+    _sdClass, 'ui',
+    useKeyOnly(left, 'left aligned'),
+    useKeyOnly(center, 'center aligned'),
+    useKeyOnly(right, 'right aligned'),
+    useKeyOnly(justified, 'justified'),
```

Comment:
Let's update these alignment classes to the same conclusion we reach [here](https://github.com/TechnologyAdvice/stardust/pull/277/files#r68423298) on the Container.